packages/cli/README.md

```diff
@@ -1,13 +1,522 @@
+
+### `deploy | d`
+
+Deploy Polywrap projects.
+
+```bash
+polywrap deploy
+```
+
+`deploy` reads the Deploy manifest (`polywrap.deploy.yaml` by default) and executes the jobs and steps listed inside.
+
+For more information on the Deploy command and the Deploy manifest, see [Configure Polywrap deployment pipeline](https://docs.polywrap.io/quick-start/build-and-deploy-wasm-wrappers/deploy-pipeline).
+
+#### Options
+- `-m, --manifest-file <path>`
+  Specify your project's manifest file.
+  By default, `deploy` searches for `polywrap.yaml`.
+
+- `-o, --output-file <path>`
+  Output file path for the deploy result
+
+### `infra | i`
+
+Modular Infrastructure-As-Code Orchestrator
+
+```bash
+polywrap infra <action> [options]
+```
+
+The `infra` command is used to set up infrastructure to test and deploy your wrappers locally.
+
+For more information on the `infra` command and how to create your own Infra modules, see [Configure Polywrap infrastructure pipeline](https://docs.polywrap.io/quick-start/test-wasm-wrappers/infra-pipeline)
+
+#### Arguments
+- `action` (required)
+  Infra allows you to execute the following actions:
+  - `up`
+    Start Polywrap infrastructure
+  - `down`
+    Stop Polywrap infrastructure
+  - `config`
+    Validate and display Polywrap infrastructure's bundled docker-compose manifest
+  - `vars`
+    Show Polywrap infrastructure's required .env variables
+
+#### Options
+- `-m, --manifest-file <path>`
+  Specify the `infra` extension manifest file.
+  By default, `infra` searches for `polywrap.infra.yaml`.
+
+- `-o, --modules <module, module>`
+  Use only specified modules
+
+#### Defaults
+
+Polywrap comes with a default `eth-ens-ipfs` module which can be used to test your wrappers locally:
+
+```
+polywrap infra up --modules=eth-ens-ipfs
+```
+
+The default infrastructure module defines a docker container with:
+
+- A test server at http://localhost:4040
+- A Ganache Ethereum test network at http://localhost:8545
+- An IPFS node at http://localhost:5001
+
+It also sets up ENS smart contracts at initialization, so you can build wrappers and deploy them to an ENS registry on your locally hosted testnet. The Ethereum address of the ENS registry is 0xe78A0F7E598Cc8b0Bb87894B0F60dD2a88d6a8Ab.
+
+### `test | t`
+
+Run Test manifests. 
+
+The `test` command executes a series of Wrapper invocations called **steps** organized into **jobs**.
+All steps within a job are run in series, while jobs are run in parallel.
+
+```bash
+polywrap run [options]
+```
+
+#### Options
+- `-m, --manifest-file <path>`
+  Specify the Workflow extension manifest file.
+  By default, `run` searches for `polywrap.test.yaml`.
+
+- `-c, --client-config <config-path>`
+  Use a custom Polywrap Client configuration.
+
+- `-o, --output-file <output-file-path>`
+  Specify the output file path for the workflow result
+
+- `-j, --jobs <jobs...>`
+  Specify ids of jobs that you want to run
+
+#### The Test manifest (`polywrap.test.yaml`)
+Basic structure:
```

Comment:
I agree that this would be much better suited to live within the `docs` repo.

Now that I look at it, https://docs.polywrap.io/quick-start/workflows/running-workflows seems outdated. Maybe rename it to `Runnning tests` and update with information found here.

What do you think?
CC @dOrgJelli @krisbitney

Reply:
That's a good idea. I think it should also be added to the manifest package readme so we can prepare for the future when our reference documentation is created by copying the readmes into the docs